qa/suites/rgw/hadoop-s3a/s3a-hadoop-v28.yaml

```diff
@@ -18,6 +18,10 @@ overrides:
         journal_collocation: true
         osd_auto_discovery: false
         journal_size: 1024
+        ceph_stable_release: luminous
+        osd_scenario: collocated
+        ceph_origin: repository
+        ceph_repository: community
```

Comment:
``ceph_repository`` should be set to ``dev`` if you're installing packages from shaman and ``ceph_dev`` is no longer valid

Reply:
@andrewschoen are you sure? This one I copied from other ceph-ansible suite which is working, I will check the logs there. ceph_dev is internal set in task.

Reply:
@vasukulkarni I'm pretty sure that if ``ceph_repository: community`` is set you'll be getting packages from download.ceph.com. I'm not surprised the job is passing, but I bet you aren't installing what you think you are. Yeah, ``ceph_dev`` isn't an option at all anymore either.

Reply:
thanks @andrewschoen , It is from download.ceph.com as you pointed out http://qa-proxy.ceph.com/teuthology/vasu-2017-09-12_20:21:30-ceph-ansible-master-distro-basic-ovh/1624063/teuthology.log and I will remove the ceph_dev key.